src/handlers/miner_dkg_handler.erl

```diff
@@ -142,8 +142,6 @@ handle_message(BinMsg, Index, State=#state{n = N, t = T,
                                 true ->
                                     case State#state.done_called of
                                         false ->
-                                            %% this needs to be a call so we know the callback succeeded so we
```

Comment:
Why was this changed?

Reply:
So I took care of this over in #537, I guess we should merge that and then rebase this to use that?